init mediawallah-enrichment-app

jeff-mw opened the pull request:
MediaWallah accepts the CLA posted in the Readme.

This is our Quickstarts Draft. We will be populating links such as sf-samples, youtube, and Blog as soon as those are approved in future PRs.

sfc-gh-lambrosetti commented:
Hi Jeff - the architecture looks great! For the quickstart, why were the package.json and package-lock.json files deleted? Once I recreated the package.json file - it was not a problem. Flagging this to @iamontheinet in case it will cause an issue during merge.

Overall, I think our team now requires notebooks for these code blocks that are in the quickstart; however, the quickstart looks pretty good.

Some notes and edits requested on the quickstart:
- The very first page could use some additional formatting. I suggested some below
- For Account / Object Level privileges, I would just tell the user they need ACCOUNTADMIN or SYSADMIN instead of listing out all the privileges. Once you have done so, I would delete these pages as it can cause some confusion.
- Use friendly names for sections instead of `01_account_setup.sql` - instead, make it *Account Setup*. the `01_account_setup.sql` is fine for cell names in the notebook.

Suggested formatting for first page:

```
**Note: The Enrichment Application is currently only available on**

`AWS US-EAST-1`

*MediaWallah is working to make this available in other regions*

MediaWallah's Enrichment Data Marketplace listing holds Two Applications. **Matchtest** and **Enrichment**.

**Matchtest** - Allows users to quickly test overlap between consumer's own data with various MediaWallah dataverse without exposure. Providing informed insights on if dataset is worth enrichment.

**Enrichment** - Returns consumer's data linked to MediaWallah's dataverse. Allowing consumers to check for data quality and usability.

Users of the free Enrichment application has a limited version of the Enrichment Application; only returning a percentage of the total for a limited time window.

Please Contact MediaWallah if additional time or trial runs are needed. Enrichment results are returned in four tables.
```

iamontheinet commented:
Don't delete package.json and/or package-lock.json. Just don't include them in your PR.

jeff-mw commented:
Hey Luke, thanks for giving it a once over. 
confirming:
* updated the first page with the changes you suggested
* removed the NN_XXX_.sql titles for standard titles
* added back in the package.json and package-lock.json

please note that this is still using the 1.3 Framework.
MW is working to upgrade to 1.6 ACF and was advised to get this one published first.
the 1.6 Quickstart will have much less code for a consumer to run as they will be installing pretty much everything using a SIDECAR runner.

please let me know if you have any questions, and if there is anything else you would like addressed. 

Thank you.

sfc-gh-lambrosetti commented:
Ok @iamontheinet - this one is good to go!

jeff-mw commented:
@sfc-gh-lambrosetti 

We also have the code in a sf-samples based on the requirements we received. But there hasn't been any movement there.
https://github.com/Snowflake-Labs/sf-samples/pull/155

iamontheinet commented on site/sfguides/src/getting-started-with-mediawallah-enrichment-native-app/getting-started-with-mediawallah-enrichment-native-app.md:
```diff
@@ -0,0 +1,959 @@
+summary: This is a guide to use MediaWallah's Enrichment Application
+categories: data-science, Getting-Started, data-science, data-applications
+environments: web
+status: Draft
```
@jeff-mw @sfc-gh-lambrosetti Change the status to Published

jeff-mw replied:
updated.

iamontheinet commented on site/sfguides/src/getting-started-with-mediawallah-enrichment-native-app/getting-started-with-mediawallah-enrichment-native-app.md:
```diff
@@ -0,0 +1,959 @@
+<!-- <img src="assets/solution_architecture.png" alt="solution_architecture" width="800"> -->
+![solution_architecture](assets/solution_architecture.png)
+
+## Setup 01: Installation
```
@jeff-mw @sfc-gh-lambrosetti Remove "Setup 01:" from the label

jeff-mw replied:
removed.

iamontheinet commented on site/sfguides/src/getting-started-with-mediawallah-enrichment-native-app/getting-started-with-mediawallah-enrichment-native-app.md:
````diff
@@ -0,0 +1,959 @@
+```
+
+<!-- ------------------------ -->
+## Step 02: Verify Installation
````
@jeff-mw @sfc-gh-lambrosetti Remove "Step 02:" from the label.

jeff-mw replied:
removed.

iamontheinet commented on site/sfguides/src/getting-started-with-mediawallah-enrichment-native-app/getting-started-with-mediawallah-enrichment-native-app.md:
```diff
@@ -0,0 +1,959 @@
+## Setup 01: Installation
+Duration: 2
+
+### Step 01: Account Setup (Post-Install)
```
@jeff-mw @sfc-gh-lambrosetti Delete this dup label

jeff-mw replied:
removed.

iamontheinet commented on site/sfguides/src/getting-started-with-mediawallah-enrichment-native-app/getting-started-with-mediawallah-enrichment-native-app.md:
```diff
@@ -0,0 +1,959 @@
+## Step 02: Verify Installation
+Duration: 1
+
+### Step 02: Verify Installation
```
@jeff-mw @sfc-gh-lambrosetti Delete this dup label.

jeff-mw replied:
removed.

iamontheinet commented on site/sfguides/src/getting-started-with-mediawallah-enrichment-native-app/getting-started-with-mediawallah-enrichment-native-app.md:
````diff
@@ -0,0 +1,959 @@
+```
+
+<!-- ------------------------ -->
+## Step 03: Onboarding
````
@jeff-mw @sfc-gh-lambrosetti Remove "Step 03:" from the label

iamontheinet commented on site/sfguides/src/getting-started-with-mediawallah-enrichment-native-app/getting-started-with-mediawallah-enrichment-native-app.md:
```diff
@@ -0,0 +1,959 @@
+## Step 03: Onboarding
+Duration: 1
+
+### Step 03: Onboarding
```
@jeff-mw @sfc-gh-lambrosetti Delete this dup label

jeff-mw replied:
removed.

iamontheinet commented on site/sfguides/src/getting-started-with-mediawallah-enrichment-native-app/getting-started-with-mediawallah-enrichment-native-app.md:
````diff
@@ -0,0 +1,959 @@
+```
+
+<!-- ------------------------ -->
+## Step 04: Log/Metric Sharing
````
@jeff-mw @sfc-gh-lambrosetti Remove "Step 04:" from the label

jeff-mw replied:
removed.

iamontheinet commented on site/sfguides/src/getting-started-with-mediawallah-enrichment-native-app/getting-started-with-mediawallah-enrichment-native-app.md:
```diff
@@ -0,0 +1,959 @@
+## Step 04: Log/Metric Sharing
+Duration: 3
+
+### Step 04: Log/Metric Sharing
```
@jeff-mw @sfc-gh-lambrosetti Delete this dup label

jeff-mw replied:
removed.

iamontheinet commented on site/sfguides/src/getting-started-with-mediawallah-enrichment-native-app/getting-started-with-mediawallah-enrichment-native-app.md:
````diff
@@ -0,0 +1,959 @@
+```
+
+<!-- ------------------------ -->
+## Step 05: Enablement
````
@jeff-mw @sfc-gh-lambrosetti Remove "Step 05:" from the label

jeff-mw replied:
removed.

iamontheinet commented on site/sfguides/src/getting-started-with-mediawallah-enrichment-native-app/getting-started-with-mediawallah-enrichment-native-app.md:
```diff
@@ -0,0 +1,959 @@
+## Step 05: Enablement
+Duration: 1
+
+### Step 05: Enablement
```
@jeff-mw @sfc-gh-lambrosetti Delete this dup label

jeff-mw replied:
removed.

jeff-mw commented:
@sfc-gh-lambrosetti @iamontheinet 
confirming updates have been applied in new commit.
